content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,173 @@
+
+For each scope (for example, `branch`), you can either select included or excluded values.
+When included values are selected, the rule is evaluated if one or more included values are part of the command context.
+When excluded values are selected, the rule is evaluated if the excluded values are not part of the command context.
+
+For example, to create a rule that is evaluated in all branches but `main` of the `example-repository` repository, you can define the following scope:
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope, it is evaluated for all values for that scope.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories.
+
+### Custom scope
+
+In addition to branch and repository, you can define custom scopes to filter rules that are evaluated for a specific CI pipeline.
+
+To add a custom scope when creating a rule:
+
+1. Click **+ Add Filter** and select **Custom Scope**.
+2. Define the scope name, for example, `team`.
+3. Define the scope's included or excluded values.
+
+Unlike the `branch` and `repository` scopes, custom scopes need to be provided in the `datadog-ci gate evaluate` command using the `--scope` option.
+For example, you can create a rule that is evaluated for the `example-repository` repository, but only when the team is `backend`:
+
+{{< img src="ci/rule_scope_example_repository_team_backend.png" alt="Rule scope for example-repository and team backend" style="width:90%;">}}
+
+The rule would be evaluated when this command is invoked in a CI pipeline of the `example-repository` repository:
+- `datadog-ci gate evaluate --scope team:backend`
+
+The rule would **not** be evaluated when the following commands are invoked instead:
+- `datadog-ci gate evaluate`, as it does not define any team.
+- `datadog-ci gate evaluate --scope team:api --scope team:frontend`, as it defines teams different from `backend`.
+
+## Manage rules
+
+To edit a Quality Gate rule, click the **Edit** icon to the right of the **Creator** avatar on the [Quality Gate Rules page][2].
+
+{{< img src="ci/edit_quality_gate_rule.png" alt="Edit a Quality Gates rule" style="width:90%;">}}
+
+To delete a Quality Gate rule, click the **Delete** icon next to the **Edit** button on the [Quality Gate Rules page][2]. Alternatively, you can edit the rule and click **Delete Rule**.
+
+{{< img src="ci/delete_quality_gate_rule.png" alt="Delete a Quality Gates rule" style="width:90%;">}}
+
+## Permissions
+
+Only users with the `quality_gate_rules_write` permission can create and edit Quality Gate rules. Users with the `quality_gate_rules_read` permission can view Quality Gate rules.
+For more information, see the [RBAC Permissions documentation][1].
+
+## Track changes in rules
+
+You can view information about who created, modified, and deleted Quality Gates rules in [Audit Trail][3].
```

Comment:
Maybe we could redirect to the audit trails docs for CI Visibility, in which we specify the query needed to filter Quality Gate events. For now it's only in [staging](https://docs-staging.datadoghq.com/rodrimendoza/adding-ci-visibility-audit-events-docs/account_management/audit_trail/events/#ci-visibility-events)

Reply:
This is what you'll do next right? Could you also update this link as part of that work? I think it would be better

Reply:
Ok, great!